contracts/mocks/Token.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
you can use the token mock already available in tests/

Reply:
Truffle has a bug that it doesn't compile `.sol` files in `tests/` dir when I specify a file in the `truffle test /file` command. That's I moved the `mocks` folder into `contracts` dir